### ISSUE TITLE
Export/Import labels

### ISSUE THREAD

**Issue by nombrekeff** [isaacs/github#1448]
**It would be nice to be able to import labels from another repository.**

I usually always setup my labels the same way. And every time I create a new repo I have to create all the labels again, one by one. 

It would be nice to be able to import labels in some way. Maybe from another repo or maybe from a file...

Let me know what you guys think about this!

**Comment by clarkbw** [comment]
Check out https://probot.github.io/apps/settings/

**Comment by nombrekeff** [comment]
Ohh cool thanks, thats exactly what I wanted!!! 
Is it a new addition? I used settings a time ago and didnt see that option

**Comment by TPS** [comment]
@nombrekeff I'd say that's a great way to get it done in GitHub, but isn't _built-in_, which is what this repo's about, so I'd like to reopen.  Objection?

**Comment by nombrekeff** [comment]
Yup, sure, I would also like seeing it built-in so yeah, I'll open it myself!

**Comment by nombrekeff** [comment]
Hey, can I ask what this `migration` label is about?
Just curious!

**Comment by nombrekeff** [comment]
I also came across [this](https://github.com/isaacs/github/issues/1279) issue, wich adds the option of having labels shared inside an organization... 
this could be a good way of accomplishing this in my opinion... at least for orgs...

**Comment by TPS** [comment]
@nombrekeff I see <kbd>migration</kbd> as problems that make migrating from/to another service/GitHub difficult.  (I'll add that in the label description.)

**Comment by nombrekeff** [comment]
Ohh I see, thanks for the explanation!

**Comment by nombrekeff** [comment]
It can now be done in `/organizations/{org}/settings/labels`, you can specify a set of labels and they will be added to all new repos.

I would suggest closing this issue, if nobody has any objection

**Comment by stefstelea** [comment]
but those are just the default ones, I think this feature would still be useful to have (migrate from repo to repo). I suggest we reopen this issue as I still find it useful

**Comment by nombrekeff** [comment]
Okay, no problem, I also think it would be useful in some cases!

**Comment by NBprojekt** [comment]
Even if I edit or create standard labels in my organization, I have to adapt every repository in which I want to make these changes myself.

This can be time consuming if you a lot repositories.

**Comment by nombrekeff** [comment]
Totally agree! I find myself in that situation quite often 😒

**Comment by rtek** [comment]
> It can now be done in `/organizations/{org}/settings/labels`, you can specify a set of labels and they will be added to all new repos.
> 
> I would suggest closing this issue, if nobody has any objection

Is there any documentation for this endpoint? Its not turning up anything in the API docs.

**Comment by nombrekeff** [comment]
@rtek Hey, I might have misspoken there, It's actually the path for https://github.com, not the API. It would be [https://github.com/organizations/{org}/settings/labels](https://github.com/organizations/{org}/settings/labels)

It's in the org settings, under labels.
